js/components/controlBar.js

```diff
@@ -327,91 +330,115 @@ var ControlBar = React.createClass({
+                 responsivenessMultiplier={this.responsiveUIMultiple} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height}
+                 text={this.props.playerState == CONSTANTS.STATE.PLAYING ? Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.PAUSE, this.props.localizableStrings) :
+                    this.props.playerState == CONSTANTS.STATE.END ? Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.REPLAY, this.props.localizableStrings) : Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.PLAY, this.props.localizableStrings) }>
+          <Icon {...this.props} icon={playIcon} style={dynamicStyles.iconCharacter}/>
+        </Tooltip>
+      </button>}).bind(this),
+
+      "live": (function(alignment){return <a className={liveClass}
         ref="LiveButton"
         onClick={liveClick} key="live">
         <div className="oo-live-circle"></div>
         <span className="oo-live-text">{liveText}</span>
-      </a>,
+      </a>}).bind(this),
 
-      "volume": <div className="oo-volume oo-control-bar-item" key="volume">
+      "volume": (function(alignment){return  <div className="oo-volume oo-control-bar-item" key="volume">
         <button className="oo-mute-unmute oo-control-bar-item"
           onClick={this.handleVolumeIconClick}
           onMouseUp={this.blurOnMouseUp}
           onMouseOver={this.highlight}
           onMouseOut={this.removeHighlight}
           tabIndex="0"
           aria-label={volumeAriaLabel}>
-          <Icon {...this.props} icon={volumeIcon} ref="volumeIcon"
+          <Tooltip enabled={isTooltipEnabled} text={this.props.controller.state.volumeState.muted ? Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.UNMUTE, this.props.localizableStrings) : Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.MUTE, this.props.localizableStrings)} 
+            responsivenessMultiplier={this.responsiveUIMultiple} bottom={this.responsiveUIMultiple * this.props.skinConfig.controlBar.height} alignment={alignment}>
+            <Icon {...this.props} icon={volumeIcon} ref="volumeIcon"
             style={this.props.skinConfig.controlBar.iconStyle.inactive} />
+          </Tooltip>
         </button>
         {volumeControls}
-      </div>,
+      </div>}).bind(this),
 
-      "timeDuration": <a className="oo-time-duration oo-control-bar-duration" style={durationSetting} key="timeDuration">
+      "timeDuration": (function(alignment){return <a className="oo-time-duration oo-control-bar-duration" style={durationSetting} key="timeDuration">
         <span>{playheadTimeContent}</span>{totalTimeContent}
-      </a>,
+      </a>}).bind(this),
 
-      "flexibleSpace": <div className="oo-flexible-space oo-control-bar-flex-space" key="flexibleSpace"></div>,
+      "flexibleSpace": (function(alignment){return <div className="oo-flexible-space oo-control-bar-flex-space" key="flexibleSpace"></div>}).bind(this),
```

Comment:
I think this extra function is a lot of code to add to each component and not all buttons need it. Please consider this other approach which stores the position of the buttons in a variable (look for references to `this.tooltipAlignments`):
[controlBar2.txt](https://github.com/ooyala/html5-skin/files/1226698/controlBar2.txt)